docs/docs/workflows/create.md

```diff
@@ -70,13 +70,27 @@ Create the `constellation-state.yaml` using the output from the Terraform state
 
 ```bash
 CONSTELL_IP=$(terraform output ip)
-CONSTELL_INIT_SECRET=$(terraform output initSecret | jq -r | tr -d '\n' | base64)
-touch constellation-state.yaml
-yq eval '.version ="v1"' --inplace constellation-state.yaml
+CONSTELL_INIT_SECRET=$(terraform output initSecret | yq -r | tr -d '\n' | base64)
```

Comment:
For the internal LB feature, I assume?

Reply:
We should probably make sure that the TF output names are aligned with the state file fields so that we can just refer to `terraform output` here and the user can trivially map them to the state file fields. Would save some explanation here

Reply:
This is still not up to date. The new output field are named `out_of_cluster_endpoint` and `in_cluster_endpoint`.

Reply:
I don't know how trivially we can change the `Infrastructure` section of the state file or if we need a migration. I'm fine either way you want to align them.

Reply:
They are already in the state file (see main branch) but be also need to document here, right?